### ISSUE TITLE
plugin not working

### ISSUE THREAD

**Issue by galapah** [ubaldot/vim-conda-activate#12]
I'm running vim in WSL2 (Windows 11, Ubuntu 22.04). I've compiled vim 9.1 with Python3 support.
I open a python script from my target conda environment being active.

I issue the command `:CondaActivate` and select the right environment (matching the one I ran the script in). I send a line to the interpreter. While environment independent commands execute properly, the environment specific ones don't work, e.g.:

`ModuleNotFoundError: No module named 'pandas_ta'`

**Comment by ubaldot** [comment]
How do you send a line to the interpreter? 
In what environment you happen to be if you run :`terminal`? Perhaps conda has changed something in newer versions…

**Comment by galapah** [comment]
Thanks, ubaldot, for your fast response!

Initially, what I was trying to do was to execute the line in a tmux terminal, but that'd be perhaps a more complex problem.

Then I tried it in the vim terminal directly (:term). It opened with the base environment. I activated my target environment but the import didn't work anyway.

Due to other (unrelated) problems with the self-compiled vim, I've switched to the vim from conda-forge and it seems to solve all my problems.
